.drone.yaml

```diff
@@ -0,0 +1,46 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: build & push
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/ukhomeofficedigital/hocs-base-image
```

Comment:
Is the repo set up on Quay?

Reply:
no, we (you 😁 ) need to set up the robot to have write permission